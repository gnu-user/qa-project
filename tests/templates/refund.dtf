05_buyer___________seller__________000010.00