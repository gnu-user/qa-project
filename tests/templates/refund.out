Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the buyer's username: 
Enter the seller's username: 
Enter the amount of credit to transfer: 
Buyer refunded successfully.